roles/technical_lead.md

```diff
@@ -1,34 +1,33 @@
 Technical Lead
 ==============
 
-## The Technical Lead is accountable for the engineering approach and code quality the team delivers 
-*An Technical Lead is a formal role that is added to an engineer’s normal responsibilities. Becoming an TL does not result in a change in title, and people can switch in and out of this role on different teams based on what is the best fit for the team and individual at that time.*
+## The Technical Lead is accountable for the engineering approach and code quality the team delivers
+
+*A Technical Lead is a formal role that is added to an engineer’s normal responsibilities. Becoming a TL does not result in a change in title, and people can switch in and out of this role on different teams based on what is the best fit for the team and individual at that time.*
 
 *Technical Lead is a leadership role. The technical success & output of their team is more important than the code they themselves write.*
 
 ### Accountabilities
 
-* *If the technical solution the team chooses doesn’t solve the problem, it’s on the TL*
-* *If the team is doing big upfront architecture rather than working out how to break features into vertical slices that add value, it’s on the TL*
-* *If the quality of the code the team is putting out isn’t good (lots of defects, low test coverage) or UX performance is slow, it’s on the TL*
-* *If the team doesn’t have good engineering practices (like CI & unit tests), it’s on the TL*
-* *If the team isn’t addressing tech debt & it’s impacting their ability to deliver it’s on the TL*
+It's the responsibility of the TL to ensure all of the following:
+
+* *The technical solution that the team chooses efficiently solves the problem*
```

Comment:
Thank you for simplifying and humanizing that `it's on the TL` phrasing 📿  as you've done here in your edit. I think the voice/tone is far more approachable now!